openadapt/events.py

```diff
@@ -249,31 +297,38 @@ def get_merged_events(to_merge, state):
         merged_event.mouse_dy = mouse_dy
         return [merged_event]
 
-
     return merge_consecutive_action_events(
-        "mouse_scroll", events, is_target_event, get_merged_events,
+        "mouse_scroll",
+        events,
+        is_target_event,
+        get_merged_events,
     )
 
 
-def merge_consecutive_mouse_click_events(events):
-    """Merge consecutive mouse click events into a single doubleclick event"""
+def merge_consecutive_mouse_click_events(events: list) -> list:
+    """Merge consecutive mouse click events into a single doubleclick event.
+
+    Args:
+        events (list): The list of events to process.
 
+    Returns:
+        list: The merged list of events.
 
-    def get_recording_attr(event, attr_name, fallback):
+    """
+
+    def get_recording_attr(event: Any, attr_name: Any, fallback: Any) -> Any:
```

Comment:
`event: ActionEvent, attr_name: str, fallback: Any`

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/d88767ff785961d7eb36ca589d96f707a019b507